CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 ### API Breaking
 * (apps) [\#8476](https://github.com/cosmos/ibc-go/pull/8476) Remove `ParamSubspace` from all `Keeper` constructors
 * (light-clients/08-wasm) [\#8511](https://github.com/cosmos/ibc-go/pull/8511) Remove deprecated `Checksums` type
+* (core/02-client) [\#8516](https://github.com/cosmos/ibc-go/pull/8516) Remove deprecated `SubmitMisbehaviour` RPC endpoint and types
```

Comment:
```suggestion
* (core/02-client) [\#8516](https://github.com/cosmos/ibc-go/pull/8516) Remove deprecated `SubmitMisbehaviour` message handler
```

I think RPC endpoint is not precise in this case. It's sort of that too, but the core of it is a message handler.